shared/message/support/nusight/Overview.proto

```diff
@@ -28,49 +28,46 @@ import "Vector.proto";
 import "Matrix.proto";
 
 message Overview {
-
-    // The timestamp this overview packet was sent
+    /// The timestamp this overview packet was sent
     google.protobuf.Timestamp timestamp = 1;
+    uint32                    robot_id  = 2;
 
-    // The ID number of this robot
-    uint32 robot_id = 2;
-
-    // The name of the binary this robot is executing
+    /// The name of the binary this robot is executing
     string role_name = 3;
 
-    // The current battery voltage as a percentage
+    /// The current battery voltage as a percentage
     float battery = 4;
 
-    // The current voltage of the battery
+    /// The current voltage of the battery
```

Comment:
In volts, I assume

Reply:
Nope. I'll add a `@see` to this comment directing the reader to the Sensors.proto field this field is based on.